### PR TITLE
Tests initial

### PR DESCRIPTION
**Pull request by monikasudol**
Some tests, review it, please :)

### CODE REVIEW

File: jest.config.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "setupFiles": [
+    "raf/polyfill",
+    "<rootDir>/src/tests/setupTests.js"
+  ],
+  "moduleNameMapper": {
+    "\\.(css|less)$": "identity-obj-proxy"
+  }
+}
```
Comment (yuriyyakym) [inline jest.config.json]: missing newline

File: src/modules/login/index.js [hunk 2]
```diff
@@ -2,16 +2,16 @@ import React, { Component } from 'react';
...
-class Login extends Component {
+export class Login extends Component {
   constructor() {
     super();
     this.state = {
       username: '',
       password: ''
     }
-  }
+  };
```
Comment (yuriyyakym) [inline src/modules/login/index.js]: redundant semicolon

File: src/modules/logo/index.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import FaChat from 'react-icons/lib/fa/comments';
-import './style.css';
+//import './style.css';
```
Comment (yuriyyakym) [inline src/modules/logo/index.js]: uncomment

File: src/modules/welcome-page/index.js [hunk 4]
```diff
@@ -4,7 +4,7 @@ import { withRouter } from 'react-router';
 import Header from '../header';
 import Login from '../login';
 import Registration from '../registration';
-import './style.css';
+//import './style.css';
```
Comment (yuriyyakym) [inline src/modules/welcome-page/index.js]: uncomment

File: src/tests/components/direct-messages.test.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+describe('Should render DirectMessages container', () => {
+  const mockStore = configureStore();
+  let store, wrapper, fetchUsers;
+
+  beforeEach(() => {
+    store = mockStore(state);
+    store.dispatch = jest.fn();
+    wrapper = shallow(
+      <ConnectedDirectMessages store={store} />)
```
Comment (yuriyyakym) [inline src/tests/components/direct-messages.test.js]: wrong formatting

File: src/tests/components/login.test.js [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+describe('Should render Login container', () => {
+  const initialState = { login: { invalidLogin: false } };
+  const mockStore = configureStore();
+  let store, wrapper, onLogIn;
+
+  beforeEach(() => {
+    store = mockStore(initialState);
+    store.dispatch = jest.fn();
+    wrapper = shallow(
+      <ConnectedLogin store={store} />)
```
Comment (yuriyyakym) [inline src/tests/components/login.test.js]: same as above

File: src/tests/components/registration.test.js [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+describe('should create Registration Container', () => {
+  const mockStore = configureStore();
+  let store, wrapper, onAddingNewUser;
+
+  beforeEach(() => {
+    store = mockStore();
+    store.dispatch = jest.fn();
+    wrapper = shallow(
+      <ConnectedRegistration store={store} />)
```
Comment (yuriyyakym) [inline src/tests/components/registration.test.js]: same here

File: src/tests/fixtures/messages.js [hunk 8]
```diff
@@ -0,0 +1,9 @@
+export default {
+  messages: [
+    { text: 'hello', timestamp: 3, userId: 0 },
+    { text: 'world', timestamp: 4, userId: 1 }
+  ],
+  loading: false,
+  id: 5,
+  conversetionist: 1
+};
```
Comment (yuriyyakym) [inline src/tests/fixtures/messages.js]: missing newline

File: src/tests/fixtures/users.js [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+    name: 'Joanna',
+    id: 0,
+    username: 'Asia'
+  },
+  {
+    name: 'Aleksandra',
+    id: 1,
+    username: 'Ola'
+  }
+];
```
Comment (yuriyyakym) [inline src/tests/fixtures/users.js]: missing newline

File: jest.config.json (hunk 1 above)
Comment (monikasudol) [reply jest.config.json]: added

File: src/modules/login/index.js (hunk 2 above)
Comment (monikasudol) [reply src/modules/login/index.js]: removed

File: src/modules/logo/index.js (hunk 3 above)
Comment (monikasudol) [reply src/modules/logo/index.js]: uncommented

File: src/modules/welcome-page/index.js (hunk 4 above)
Comment (monikasudol) [reply src/modules/welcome-page/index.js]: uncommented

File: src/tests/components/direct-messages.test.js (hunk 5 above)
Comment (monikasudol) [reply src/tests/components/direct-messages.test.js]: improved

File: src/tests/components/login.test.js (hunk 6 above)
Comment (monikasudol) [reply src/tests/components/login.test.js]: same as above

File: src/tests/fixtures/messages.js (hunk 8 above)
Comment (monikasudol) [reply src/tests/fixtures/messages.js]: done

File: src/tests/fixtures/users.js (hunk 9 above)
Comment (monikasudol) [reply src/tests/fixtures/users.js]: done

**Pull request merged**